ui/src/app/pages/admin/admin-user-bypass.tsx

```diff
@@ -68,12 +68,16 @@ export class AdminUserBypass extends React.Component<
 
   render() {
     const {selectedModules, open} = this.state;
+    const {enableBetaAccess,
+      enableComplianceTraining,
+      enableEraCommons,
+      enableDataUseAgreement} = serverConfigStore.getValue();
     return <PopupTrigger
         side='bottom'
         onClose={() => {this.cancel(); this.setState({open: false}); }}
         onOpen={() => this.setState({open: true})}
         content={<FlexColumn style={{padding: '1rem'}}>
-          <FlexRow style={{justifyContent: 'space-between'}}>
+          {enableBetaAccess && <FlexRow style={{justifyContent: 'space-between'}}>
```

Comment:
nit: probably group this and the next element into a `<React.Fragment>` so you just have a single conditional check. Example: https://github.com/all-of-us/workbench/blob/676d65fd44f9ff9fc09f599fdb204b84330630a8/ui/src/app/pages/profile/data-user-code-of-conduct.tsx#L130-L152